packages/manager/modules/pci/src/components/project/regions-list/regions-list.controller.js

```diff
@@ -49,14 +74,47 @@ export default class RegionsListController {
     );
   }
 
-  updateRegions() {
-    const formattedRegions = map(this.regions, (region) => ({
-      ...this.ovhManagerRegionService.getRegion(region.name),
-      name: region.name,
-      continentCode: region.continentCode,
-      hasEnoughQuota: region.hasEnoughQuota(),
-      isLocalZone: region.isLocalZone,
-    }));
+  async updateRegions() {
+    if (this.deploymentMode) {
+      await this.PciProjectStorageContainersService.getProductAvailability(
```

Comment:
No issue with using async/await in $OnInit ?

Reply:
Good point. Prefer using `.then` promise to avoid issues with `$digest` lifecycle